components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/AbstractJSObjectWrapper.java

```diff
@@ -26,9 +26,13 @@
  * @param <T> Wrapped object type
  */
 public abstract class AbstractJSObjectWrapper<T> extends AbstractJSObject {
-    protected T wrapped;
+    private T wrapped;
 
     public T getWrapped() {
         return wrapped;
     }
+
+    protected void setWrapped(T wrapped) {
```

Comment:
We can use the constructor. 
Then we can guarantee that it is not null.
Then we can get rid of null checks.